ui/selectmenu.js

```diff
@@ -603,7 +604,7 @@ return $.widget( "ui.selectmenu", {
 		// we always remove classes first and add them second, otherwise if both classes have the
 		// same theme class, it will be removed after we add it.
 		this._removeClass( this.button, "ui-selectmenu-button-" +
-				( this.isOpen ? "closed" : "open" ) )
+			( this.isOpen ? "closed" : "open" ) )
```

Comment:
This is not part of this PR. The widget already adds these classes https://github.com/jquery/jquery-ui/blob/master/ui/selectmenu.js#L42 Also open here is correct because its a state ( adjective ) not an action ( verb ). Oddly closed is the opposite in english and closed is an adjective and close is a verb. So while it may seem inconsistent the current class names are grammatical correct.